Nybble/manifest.json

```
@@ -0,0 +1,56 @@
+{
+  "configuration": {
+    "title": "NybbleConfiguration",
+    "type": "object",
+    "properties": {
+      "nhub_url": {
+        "title": "Nhub Url",
+        "description": "Nybble Hub Connector Base URL",
+        "minLength": 1,
+        "maxLength": 2083,
+        "format": "uri",
+        "type": "string"
+      },
+      "nhub_username": {
+        "title": "Nhub Username",
+        "description": "Nybble Hub Connector username",
+        "type": "string"
+      },
+      "nhub_key": {
+        "title": "Nhub Key",
+        "description": "Nybble Hub Connector Key to authenticate the requests",
+        "type": "string"
+      },
+      "sekoia_url": {
+        "title": "Sekoia Url",
+        "description": "Sekoia.io Base URL",
+        "minLength": 1,
+        "maxLength": 2083,
+        "format": "uri",
+        "type": "string"
+      },
+      "sekoia_api_key": {
+        "title": "Sekoia Api Key",
+        "description": "Sekoia.io API Key",
+        "type": "string"
+      }
+    },
+    "required": [
+      "nhub_url",
+      "nhub_username",
+      "nhub_key",
+      "sekoia_url",
+      "sekoia_api_key"
+    ],
+    "secrets": [
+      "nhub_key",
+      "sekoia_api_key"
+    ]
+  },
+  "description": "[Nybble Hub](https://nybble-security.io) is the worldwide first blue team community which handles your alerts at a glance.",
+  "name": "Nybble",
+  "slug": "nybble",
+  "uuid": "0664bc2d-8f09-410f-ab8f-e0e07911a6f4",
+  "docker": "sekoia-automation-module-nybble",
+  "version": "1.0"
```

We expect that the number version follows the [semver](https://semver.org/spec/v2.0.0.html) convention.
```suggestion
  "version": "1.0.0"
```